magento/magento2
# Fix #24091 - Selected configurable product attribute options are not displaying in wishlist page.

### Description (*)

This fixes #24091 and #22503
The reason of the issue is that in add-to-wishlist.js there are event observers that are being triggered on for every configurable product on page after modifying one of them. So if the last product on page doesn't contain any option checked, no matter which will be added to wishlist, it won't contain options.

### Related Pull Requests


### Fixed Issues (if relevant)

Fixes #22503
Fixes #24091 

### Manual testing scenarios (*)

Check fixed issues for steps to test

1) Created two Configurable Product 
2) The born products must be with one or more 'VisualSwatchOption', and related to one category.
3) Go to Category Page
4) Select 'VisualSwatchOption' for a first product in the list category and add to Wishlist
5) For added product must be present 'See Details' on Wishlist page

### Questions or comments


### Contribution checklist (*)
